javascript/javascript1/week1/pizza-exercise.js

```diff
@@ -0,0 +1,16 @@
+console.log("I love pizza");
+let pizzaName = "chicken-pizza";
+let pizzaPrice =10;
+let noOfPizza = 2;
+let familySize = true;
+let totalPrice;
+
+if (familySize) {
+    totalPrice= pizzaPrice*2*noOfPizza;
+}
+
+else{
+    totalPrice= pizzaPrice*noOfPizza;
+}
```

Comment:
Again consistent style, because it eases reading the code by humans (both yourself and you fellow developers).

In some of the other files you  styled `else` like so:
```
else
{
    ... // code here
}
```
which is inconsistent with this file.

Reply:
good practice to choose one consistent style. it is also done.